### DISCUSSION TITLE
Git commit log bullets?

### DISCUSSION THREAD

**Discussion by yawaramin** [ghostty-org/ghostty#3448] [Q&A] [answered]
Hi, I'm trying out Ghostty (first-time user with 1.0) and I seem to remember coming across a post at some point that showed nice-looking git commit graphs, with fancy rendered bullets instead of just normal asterisk characters, and rounded graph corners.

Am I just imagining this? Can't seem to find anything about it now.

**Comment by yochem** [comment, marked as answer]
I think you're referring to this tweet for Mitchell: https://x.com/mitchellh/status/1854586330816401877?s=46

**Reply by yawaramin** [reply]
Excellent, thanks!

**Reply by yawaramin** [reply]
According to a reply in that thread it's available only as vim/NeoVim plugins, not for the `git log --graph ...` command output: https://github.com/rbong/vim-flog

I might try it later!

**Reply by yochem** [reply]
Correct, sadly not. I also still need to try it!

**Comment by yawaramin** [comment]
Oh, I just found out that the [tig](https://jonas.github.io/tig/) TUI also draws very nice git log graphs when the config is set to `line-graphics = utf-8`.

<img
  width="1079"
  alt="Screenshot showing the tig TUI tool rendering a git log graph with fancy UTF-8 bullet and line characters in the Ghostty terminal"
  src="https://github.com/user-attachments/assets/b2e9eb21-84b0-4543-b3ed-5fc2cd831447">
